### PR TITLE
feat: update web-ext for chrome debugging

### PR DESCRIPTION
**Pull request by SunriseFox**

### CODE REVIEW

**Comment by Jack-Works** [comment]
@SunriseFox there is conflict in yarn.lock

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "start": "react-app-rewired start",
-    "start:debug": "node --inspect-brk node_modules/react-app-rewired/scripts/start.js",
+    "start:debugging": "node --inspect-brk node_modules/react-app-rewired/scripts/start.js",
```
Comment (Artoria2e5) [inline package.json]: NO.

Comment (Jack-Works) [reply package.json]: Reason?

Comment (SunriseFox) [reply package.json]: `start:debug` and `build:debugging` is magic

Comment (neruthes) [reply package.json]: What an amazing rename! I am eager to learn more about the story behind..

Comment (yisiliu) [reply package.json]: what's the point here?

Comment (Jack-Works) [reply package.json]: It's a neutral change. Why block on this? @Artoria2e5

Comment (Artoria2e5) [reply package.json]: I am not blocking on this. I am just saying that changing this to a longer, harder-to-type thing without explaining any magic is arbitrary and should not be in this PR.

**Comment by Jack-Works** [comment]
closed by https://github.com/DimensionDev/Maskbook/pull/338

**Comment by Jack-Works** [comment]
@SunriseFox can you rebase this branch?

**Comment by SunriseFox** [comment]
appended to #328

**Pull request closed without merging**